experimental/packages/exporter-logs-otlp-proto/README.md

```diff
@@ -0,0 +1,110 @@
+# An OTLP exporter to send logs using protobuf over HTTP
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+**Note: This is an experimental package under active development. New releases may include breaking changes.**
+
+This module provides exporter for logs in node/web to be used with OTLP (`http/protobuf`) compatible receivers.
+Compatible with [opentelemetry-collector][opentelemetry-collector-url] versions `>=0.32 <=0.50`.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/exporter-logs-otlp-proto
+```
+
+## Service Name
+
+The OpenTelemetry Collector Exporter does not have a service name configuration.
+In order to set the service name, use the `service.name` resource attribute as prescribed in the [OpenTelemetry Resource Semantic Conventions][semconv-resource-service-name].
+To see documentation and sample code for the logs exporter, see the [exporter-metrics-otlp-proto package][logs-exporter-url]
```

Comment:
This exporter never had this feature, so we can leave some of this out.
Also the documentation there was meant to point toward the other exporters :slightly_smiling_face: 

```suggestion
## Further Documentation

To see documentation and sample code for the traces exporter, as well as instructions for using TLS, see the [exporter-trace-otlp-proto package][trace-exporter-url].
To see documentation and sample code for the metric exporter, see the [exporter-trace-otlp-proto package][metrics-exporter-url].
```

(see suggestion below to make the links work)